features/networking/egress-ip.feature

```diff
@@ -121,12 +121,11 @@ Feature: Egress IP related features
   @destructive
   Scenario: Should remove the egressIP from the array if it was not being used
     Given I store a random unused IP address from the reserved range to the clipboard
```

Comment:
This fails for me when there is an error message on stderr, because we chomp on `@result[:response]` which combines stdout and stderr.


```
      [17:48:06] INFO> Shell Commands: oc exec sdn-546bz  --kubeconfig=/home/rbrattai/Documents/deployments/127342/workdir/ocp4_admin.kubeconfig -n openshift-sdn  --container=sdn -i -- bash -c ip\ -4\ -brief\ a\ show\ \"ens5\"\ \|\ awk\ \'\{print\ \$3\}\'
      10.0.136.130/19
      
      STDERR:
      E1215 12:48:06.767680  392724 v2.go:105] write tcp 172.16.16.104:57442->3.97.80.33:6443: use of closed network connection
      
      [17:48:06] INFO> Exit Status: 0
      [17:48:06] INFO> Subnet range for the primary interface on the node is stored in the subnet_range clipboard.
      invalid address: 19
    
```
This fails because we are trying `IPAddr.new("10.0.136.130/19\n\nSTDERR:\nE1215 18:25:29.802564  488056 v2.go:105] read /dev/stdin: resource temporarily unavailable").to_range.to_a.shuffle.each`




We need to fix the step to use `cb[cb_name] = @result[:stdout].chomp` and we also don't need the extra string interpolation for `cb.subnet_range`


```diff
diff --git a/features/step_definitions/networking.rb b/features/step_definitions/networking.rb
index f42843ac..2222c60e 100644
--- a/features/step_definitions/networking.rb
+++ b/features/step_definitions/networking.rb
@@ -528,7 +528,7 @@ Given /^I store a random unused IP address from the reserved range to the#{OPT_S
   cb_name = "valid_ip" unless cb_name
   step "the subnet for primary interface on node is stored in the clipboard"

-  reserved_range = "#{cb.subnet_range}"
+  reserved_range = cb.subnet_range

   unused_ips=[]
   #Save four unused ip in the clipboard
@@ -827,7 +827,7 @@ Given /^the subnet for primary interface on node is stored in the#{OPT_SYM} clip
     "| bash | -c | ip -4 -brief a show \"<%= cb.interface %>\" \\| awk '{print $3}' |"
   )
   raise "Failed to get the subnet range for the primary interface on the node" unless @result[:success]
-  cb[cb_name] = @result[:response].chomp
+  cb[cb_name] = @result[:stdout].chomp
   logger.info "Subnet range for the primary interface on the node is stored in the #{cb_name} clipboard."
 end
```

Reply:
@rbbratta Thanks a lot!  Learned a lot from you!